testdata/cases/proto-json-name-ignore/test.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$ref": "#/$defs/.tests.Values",
+  "$defs": {
+    ".tests.Values": {
+      "properties": {
+        "foo_bar_baz": {
+          "$ref": "#/$defs/.tests.Values.foo_bar_baz"
+        },
+        "implicit_camel_case": {
+          "$ref": "#/$defs/.tests.Values.implicit_camel_case"
+        },
```

Comment:
이거 camel case 여야 하는거 아니에요?

Reply:
필드 이름만 camel_case에요.  테스트케이스가 ignore json name이기때문에 snake case가 되는거에요.

Reply:
1. respect json_name : snake_case를 camelCase로 바꾸기
2. ignore json_name: snake_case를 유지하기

Reply:
이런 이름이 된 이유: https://protobuf.dev/programming-guides/json/#json-options

이 문서 json-options 3번 항목에 나와요. `Use proto field name instead of lowerCamelCase name`